generator/transit_generator.cpp

```diff
@@ -111,17 +129,71 @@ struct IsSortedVisitor
   bool m_isSorted = true;
 };
 
-/// \returns ref to a stop at |stops| by |stopId|.
-Stop const & FindStopById(vector<Stop> const & stops, StopId stopId)
+template <typename T>
+void Append(vector<T> const & src, vector<T> & dst)
+{
+  dst.insert(dst.end(), src.begin(), src.end());
+}
+
+void LoadBorders(string const & dir, TCountryId const & countryId, vector<m2::RegionD> & borders)
```

Comment:
But TCountryId is a string, therefore this code is almost the same as the old code? Type checker won't warn us if, for example, someone will pass a path to a country file instead of country id.

Reply:
Up to you, of course, but, probably, you still remember the hell with mwm names two years ago, that caused creation of LocalCountryFile and CountryFile abstractions. The purpose of LocalCountryFile and Countryile is to somehow wrap exact path to a file on disk, and allows to get full path so specified country indices or to the corresponding mwm file.

Reply:
I still remember that API, with "string const & countryId" parameters, API that sometimes requires file name with ".mwm" suffix, sometimes requires full path, sometimes requires file name without ".mwm" suffix, etc.

Reply:
After discussion we decided to create a class which we'll be used instead of TCountryId. It'll contain a string with country id but it has some checks to be sure it's not a path or name with expansion. But not in this PR.